ibc-core/ics03-connection/types/src/connection.rs

```diff
@@ -392,8 +391,7 @@ impl TryFrom<RawCounterparty> for Counterparty {
                 .prefix
                 .ok_or(ConnectionError::MissingCounterparty)?
                 .key_prefix
-                .try_into()
-                .map_err(|_| ConnectionError::Client(ClientError::EmptyPrefix))?,
```

Comment:
Should we also remove the `ClientError::EmptyPrefix` variant?

Reply:
[df102af](https://github.com/cosmos/ibc-rs/pull/1274/commits/df102af5df6baa4ad76af1009e95e560e0554a70)